src/lib.rs

```diff
@@ -124,6 +124,40 @@ pub fn load_native_certs() -> CertificateResult {
     }
 }
 
+/// Load certificates from the paths.
+///
+/// If both are `None`, return `Ok(None)`.
```

Comment:
Apologies, I think this is a preexisting docs issue, but I think this should be 

```suggestion
/// If both are `None`, returns an empty [`CertificateResult`].
```